nova_plugin/host_aggregate.py

```diff
@@ -86,7 +93,8 @@ def create(nova_client, args, **kwargs):
     )
 
     host_aggregate = nova_client.aggregates.create(**host_aggregate_dict)
-    _add_hosts(ctx, nova_client, host_aggregate, kwargs)
+    hosts = get_property(ctx, HOSTS_PROPERTY, kwargs, [])
```

Comment:
@kbijakowski are you trying to get `HOSTS_PROPERTY` from yaml template? if so, I can't see `hosts` added to `create` in `plugin.yaml`